src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml

```diff
@@ -89,7 +89,7 @@
                        Style="{StaticResource SettingsGroupTitleStyle}"
                        Margin="{StaticResource XSmallBottomMargin}"/>
 
-            <TextBlock Text="Version 0.15.2"
+            <TextBlock Text="{x:Bind ViewModel.PowerToysVersion }"
```

Comment:
for consistency purposes, I suggest you use the static view model resource as in other controls within the file.